easybuild/easyconfigs/c/CP2K/CP2K-svn-CrayGNU-2016.11-cuda-8.0.54.eb

```diff
@@ -0,0 +1,59 @@
+# contributed by Luca Marsella (CSCS)
+easyblock = 'MakeCp'
+
+name = 'CP2K'
+version = "svn"
```

Comment:
Right now I was not planning to build it with Jenkins for the general public in production (it is not a stable official release), but just send it to two users to fix their bug. I hope the CP2K developers will soon publish a new stable release with the bug fix.